### ISSUE TITLE
Getting ejected from website, need to accept discord again

### ISSUE THREAD

**Issue by Dejnaa** [Icecrown-Community/ICC-Member-Website-Public#81]
During my 3 curve shifts this week I had 1 reoccurring problem

Every 10-15min I was suddenly sent out from the raid (clicked reload as I do every 2-3 min) back to the start of the website where it says this website is only for members if ICC, then you have to accept to log in with your discord credentials. After clicking yes, I am sent to home page - have to click on Goldcollecting, Select the raid and then I am back. 

Add to that that the loading is not fast considering how many people are currently trying to book customers. This gives not only annoyance but also stress, as it disrupts the work I do when time is of the essence. 

Can you please looking into why this is happening? Or if you know why, fix it? (I am using Chrome if that has any correlation) 
I can stream it next time if you want

**Comment by iElsha** [comment]
Did that happen again? @Dejnaa

**Comment by Dejnaa** [comment]
Last Curve run I only got thrown out 1 time I think - have another one on Friday and will see then :)

**Comment by iElsha** [comment]
Did you get a toast message at the bottom?
Did that happen randomly and after having clicked a button?

**Comment by Dejnaa** [comment]
Yes, when I get logged out a red message showed up in the bottom, I didnt take a printscreen and the message wasent to obvious.
If I see it again I will take a printscreen. Tomorrow I have the 14:00 curve shift, so lets see if it happens again since you have fixed some issues with the long loading ect the past week. After tomorrow if I dont experience that anymore we can close this ticket :)

**Comment by Dejnaa** [comment]
Hey! This do not happen anymore, thank you :)

**Comment by Wanhedaa** [comment]
@iElsha 
> Every 10-15min I was suddenly sent out from the raid (clicked reload as I do every 2-3 min) back to the start of the website where it says this website is only for members if ICC, then you have to accept to log in with your discord credentials. After clicking yes, I am sent to home page - have to click on Goldcollecting, Select the raid and then I am back.

i had the same problem - at advertising panel - yesterday, after 20 min it just kicked me and wanted the authentication again. After i clicked on everything, it took me back to the homepage, and had to find the raid what i was waiting for

**Comment by Fakevox** [comment]
Can also confirm was happening to me during my last curve shift.

**Comment by iElsha** [comment]
We need the error message in the toast on the bottom if possible, will be easier for us

**Comment by Fakevox** [comment]
Something about JSON from recollection will screenshot next time.

**Comment by Wanhedaa** [comment]
@iElsha 
![image](https://user-images.githubusercontent.com/77633500/108625576-f2371f00-744b-11eb-9773-f4a305cd4ba4.png)

**Comment by Fakevox** [comment]
Can confirm this is also the message I regularly see.

**Comment by Wanhedaa** [comment]
![image](https://user-images.githubusercontent.com/77633500/108866357-6e6c6680-75f4-11eb-912f-ee08533c5b3d.png)
Another one, i just got dropped from the web while doing curve collection

**Comment by Dejnaa** [comment]
Elsha, I still get this, but much less frequent.
![image](https://user-images.githubusercontent.com/77668716/114276686-0b118900-9a28-11eb-9895-d77c6125ba70.png)

Last times I have noticed it happens when the time is past the raidtime 
Could be that the RL did change the status of the raid aswell - so that might help you recreate the "issue"

**Comment by iElsha** [comment]
We have someone on this. In the meantime, I've extended the auth max-age to 14 days instead of 3 days.

**Comment by Dejnaa** [comment]
I have not had any issues with this for a while now - should I close this ticket? :)

**Comment by Wanhedaa** [comment]
I'm still using it on every day due to curves, and i still get ejected with the same error. Don't close it yet.

**Comment by Wanhedaa** [comment]
![image](https://user-images.githubusercontent.com/77633500/122787010-2fee6c00-d2b5-11eb-89d8-c063edac922c.png)
I'm still getting ejected from the web while doing curve collectings, today it did 2  times in 2 hours.

**Comment by Wanhedaa** [comment]
update: We were watching the web with Moodwend together and we both got ejected at the same time. Maybe can be a helpfull info

**Comment by Dejnaa** [comment]
I am not sure if this happens much anymore. I think I got ejected like 1 time the last 2 weeks.

**Comment by Wanhedaa** [comment]
you are not using the website as often and as long as the curve gold collectors / active advertisers. It is still a thing and it is still annoying.

**Comment by Dejnaa** [comment]
Thats why I bumped it :) Good to know it still makes problems
